### ISSUE TITLE
DiscordAPIError[50001]: Missing Access at `/rest`

### ISSUE THREAD

**Issue by MaksiRose** [MaksiRose/paw-and-paper#617]
Created by: maksi#8022 (268402976844939266)

```
DiscordAPIError[50001]: Missing Access
    at SequentialHandler.runRequest (/home/discord-bots/paw-and-paper/node_modules/@discordjs/rest/dist/index.js:743:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async SequentialHandler.queueRequest (/home/discord-bots/paw-and-paper/node_modules/@discordjs/rest/dist/index.js:549:14)
    at async REST.request (/home/discord-bots/paw-and-paper/node_modules/@discordjs/rest/dist/index.js:988:22)
    at async MessageManager.edit (/home/discord-bots/paw-and-paper/node_modules/discord.js/src/managers/MessageManager.js:175:15)
    at async Timeout._onTimeout (/home/discord-bots/paw-and-paper/dist/commands/gameplay_maintenance/rest.js:109:13)
{
	"id": "1025284583568195584",
	"application_id": "862718885564252212",
	"type": 2,
	"data": {
		"id": "1021101571993509937",
		"name": "vote",
		"type": 1,
		"options": []
	},
	"guild_id": "1009121624701796504",
	"channel_id": "1019993887223320616",
	"user_id": "723329476142039130",
	"timestamp": 1664517293589,
	"app_permission": "1072503843649",
	"locale": "en-US",
	"guild_locale": "en-US"
}
```

Additional conversation:
```
FULL CONVERSATION REGARDING THIS TICKET

ADMIN: Hello Eris! I‘m the developer behind Paw and Paper. I‘m writing you because of your bug report. First off: As it says on the bottom of each error message, please only submit the same error message once. Otherwise, it just spams me with bug reports unnecessarily. Thank you! :)
Furthermore, the error message suggests that the bot is trying to edit one of its messages, but doesn‘t have access to it. Are you sure that the bot has the necessary permissions and is in the memberlist of the channel that you are using the bot in? If you are using it in a thread, try pinging the bot. This will add it to the thread.
To reply, please use the button attached to this message. Messages sent directly into the DMs are not recorded for privacy purposes.

USER: ah crap sorry about the spam-- 

ADMIN: No worries! Just let me know if you are able to resolve the issue and if not, I would love to know some more context (What invite did you use to invite the bot, did you change the permissions in any way, is it a thread or a normal channel, does the bot appear in the memberlist of the channel, etc)

USER: just a regular channel, i was using the /vote command i voted then the bot sent a bunch of messages replying to the /vote message, my first thought was delete the message then it wont be able to reply and possibly correct its self, then it continued to reply to a message that no longer existed, after that i pushed the red button thing a bunch of times like a dummy...

ADMIN: Thanks for your response!! That‘s very strange behavior. Would you be so kind and maybe upload an image or a video showing how it looks? You can just upload it into the chat first, then copy the link and paste it after clicking the reply button :)

USER: https://discord.com/channels/@me/1021095931233435739/1025310161935532102

ADMIN: Hi, sadly you just pasted the link to the message, not the picture/video in the message. I can't open the message since it's a DM message ^^ To copy the link of the media, click on it to make it fullscreen, on mobile you should see a button on the top right that can copy it, and on pc you can right click and then click Copy Link.

USER: https://media.discordapp.net/attachments/1021095931233435739/1025310161704865812/Screenshot_20220930-033545.png

https://media.discordapp.net/attachments/1021095931233435739/1025310160924708895/Screenshot_20220930-033535.png

https://media.discordapp.net/attachments/1021095931233435739/1025310160723386428/Screenshot_20220930-033521.png

https://media.discordapp.net/attachments/1021095931233435739/1025310160463351860/Screenshot_20220930-033507.png

ADMIN: Thank you for all your help! Together with the logs I was able to figure out exactly what happened:
One feature of Paw and Paper is that after 10 minutes of inactivity, you will automatically start resting. This is useful because otherwise if you forget to rest yourself, you will have little energy next time you want to play and that would be frustrating.
Something must've went wrong when sending the original message telling you that you are now resting, because after that it tried to edit a message that didn't exist or that the bot doesn't have access to for some other reason. Because of the error, the loop never got a chance to stop, and therefore it kept showing you more errors.
TLDR; I know how to fix the bug now. Thank you for  your help!
```
